pac4j-saml/src/main/java/org/pac4j/saml/client/Saml2Client.java

```diff
@@ -92,12 +73,9 @@
  * @author Michael Remond
  * @since 1.5.0
  */
-public class Saml2Client extends BaseClient<Saml2Credentials, Saml2Profile> {
-
-    protected static final Logger logger = LoggerFactory.getLogger(Saml2Client.class);
+public class SAML2Client extends BaseClient<Saml2Credentials, Saml2Profile> {
```

Comment:
I think it would have been better to keep the name `Saml2Client` for backward compatibility.